base/linalg/eigen.jl

```diff
@@ -27,6 +27,45 @@ function getindex(A::Union{Eigen,GeneralizedEigen}, d::Symbol)
     throw(KeyError(d))
 end
 
+eigsortby(λ::Real) = λ
+eigsortby(λ::Complex) = reim(λ)
+
+"""
+    sort!(F::Base.LinAlg.Eigen; kw...)
+
+Sort the eigenvectors and eigenvalues of an eigenfactorization `F` in place using the eigenvalues
+for comparison.  See also `sort!(::AbstractVector)`.
```

Comment:
Why point to `sort!(::AbstractVector)`? Other docstrings for `sort!` will be listed in the same place anyway.

Reply:
Unless you do something like `?> sort!(F)` to filter out all the noise. 🙂 Or in the manual.

Reply:
Even then, it's not clear what this reference means: does this function support all keyword arguments accepted by `sort!(::AbstractVector)`? Is it just that they are similar? Also, the generic docstring is defined for any `v`, not just `::AbstractVector`.

Reply:
You are right. It is because it accepts the same keyword arguments. I will change the wording to make it more clear.